SafeSurroundings/SafeSurroundings/Controllers/ProfileController.cs

```diff
@@ -0,0 +1,122 @@
+﻿using SafeSurroundings.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using SafeSurroundings.Data.Services;
+using SafeSurroundings.Data.Models;
+using SafeSurroundings.Models;
+using Newtonsoft.Json;
+
+namespace SafeSurroundings.Controllers
+{
+    public class ProfileController : BaseController
+    {
+
+        InMemoryProfileTable inMemoryAccountTable;
+        InMemoryProfileTable inMemoryProfileTable;
+        InMemoryPlaceTable inMemoryPlace;
+        InMemoryMeetUpTable InMemoryMeetUpTable;
+
+        public ProfileController(InMemoryProfileTable inMemoryAccountTable, InMemoryProfileTable inMemoryProfileTable, InMemoryMeetUpTable inMemoryMeetUpTable, InMemoryPlaceTable inMemoryPlaceTable)
+        {
+            this.inMemoryAccountTable = inMemoryAccountTable;
+            this.inMemoryProfileTable = inMemoryProfileTable;
+            this.InMemoryMeetUpTable = inMemoryMeetUpTable;
+            this.inMemoryPlace = inMemoryPlaceTable;
+        }
+
+        [HttpGet]
+        [UserAuthentication]
+        public override ActionResult Index()
+        {
+            try
+            {
+                ProfileViewModel profileViewModel = new ProfileViewModel();
+                Profile currentProfile = new Profile();
+                currentProfile = (Profile)Session["profile"];
+                //Add object reflection dll
```

Comment:
Something to add to make the data.model to web.viewmodel easier. Plan for a future branch to put that in.